File: scripts/find_quotes.py
import re

path = r'c:\Users\Xgm\Desktop\APPWBP\scripts\blog-articles-productivity.mjs'
with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

quotes = re.findall(r'\"([^\"]{30,})\"', content)
for q in quotes:
    print('Quote:', q)

File: scripts/find_quotes.js
const fs = require('fs');
const path = require('path');

const filePath = path.join('c:', 'Users', 'Xgm', 'Desktop', 'APPWBP', 'scripts', 'blog-articles-productivity.mjs');
const content = fs.readFileSync(filePath, 'utf-8');

const regex = /"([^"]{30,})"/g;
let match;
while ((match = regex.exec(content)) !== null) {
  console.log('Quote:', match[1]);
}
